pkg/controllers/clusterinfo/autodetect_controller.go

```diff
@@ -85,6 +86,13 @@ func (r *AutoDetectReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		needUpdate = true
 	}
 
+	if clusterInfo.Status.DistributionInfo.Type == clusterinfov1beta1.DistributionTypeOCP && clusterInfo.Status.DistributionInfo.OCP.Version != "" {
```

Comment:
it needs ut. And i think we already collect it in cluster claim? why not update the label based on claim?